docs/getting-started-guides/vagrant.md

```diff
@@ -40,7 +40,7 @@ curl -sS https://get.k8s.io | bash
 
 The `KUBERNETES_PROVIDER` environment variable tells all of the various cluster management scripts which variant to use.  If you forget to set this, the assumption is you are running on Google Compute Engine.
 
-By default, the Vagrant setup will create a single kubernetes-master and 1 kubernetes-minion. Each VM will take 1 GB, so make sure you have at least 2GB to 4GB of free memory (plus appropriate free disk space). To start your local cluster, open a shell and run:
+By default, the Vagrant setup will create a single kubernetes-master and 1 node (kubernetes-minion). Each VM will take 1 GB, so make sure you have at least 2GB to 4GB of free memory (plus appropriate free disk space). To start your local cluster, open a shell and run:
```

Comment:
Replace first sentence with:
`By default, the Vagrant setup will create a single master node (called kubernetes-master) and 1 node (called kubernetes-minion-1).`